lib/parser.js

```diff
@@ -14,12 +14,19 @@ function parser (data, opts) {
 
   return result
 
+  function parseTty (chunk, enc, cb) {
+    this.payload.append(chunk)
+
+    publish.bind(this)()
```

Comment:
this will be very very slow. Would you mind attaching `publish` to `result`, and calling it straight from there?